src/dataviews/dataview-model-base.js

```diff
@@ -185,6 +185,7 @@ module.exports = Model.extend({
   // Class props
   {
     ATTRS_NAMES: [
+      'type',
```

Comment:
@viddo I place on dataviewModelBase because is a common attrbitute, like `enabled`.